ui/app/pages/swaps/view-quote/view-quote.js

```diff
@@ -281,9 +282,23 @@ export default function ViewQuote() {
     }
   }, [originalApproveAmount, approveAmount])
 
-  const showWarning =
+  const showInsufficientWarning =
     (balanceError || tokenBalanceNeeded || ethBalanceNeeded) && !warningHidden
 
+  const shouldShowPriceDifferenceWarning =
+    !showInsufficientWarning &&
+    ['high', 'medium'].includes(usedQuote?.priceSlippage?.bucket)
+  let priceDifferencePercentage = 0
+  if (usedQuote?.priceSlippage?.ratio) {
+    priceDifferencePercentage = parseFloat(
```

Comment:
Any reason the parseFloat is still included?

Reply:
Yep!  The `parseFloat` removes excess 0's in the decimal places.  I don't want to show `8.00%`, simply `8%`; or, `8.10` becomes `8.1`

Reply:
ah, nice